ote_sdk/ote_sdk/test_suite/training_tests_actions.py

```diff
@@ -562,15 +562,23 @@ def __call__(self, data_collector: DataCollector, results_prev_stages: OrderedDi
 
 
 # TODO: think about move to special file
-def check_nncf_model_graph(model, path_to_dot):
+def check_nncf_model_graph(model, path_to_dot, force_regen_dot=False):
```

Comment:
Here as well. Would be good to know the arg and return types, which improves the readability.

Reply:
done